docs/rules/generator-star-spacing.md

```diff
@@ -137,6 +158,46 @@ var anonymous = function*() {};
 var shorthand = { *generator() {} };
 ```
 
+Examples of **correct** code for this rule with overrides present:
+
+```js
+/*eslint generator-star-spacing: ["error", {
+    "before": false,
+    "after": true,
+    "anonymous": "neither",
+    "method": {"before": true, "after": true}
+}]*/
+/*eslint-env es6*/
+
+function* generator() {}
+
+var anonymous = function*() {};
+
+var shorthand = { * generator() {} };
+
+class Class { static * method() {} }
+```
+
+Examples of **incorrect** code for this rule with overrides present:
```

Comment:
Very minor nitpick: In most of the documentation, incorrect code is listed before correct code. It would be nice to also do that here.